### ISSUE TITLE
FriendlyName not supported in Unix

### ISSUE THREAD

**Issue by ferantivero** [dotnet/corefx#29061]
it might be worth to mention from the [Cross-Platoform Cryptography doc](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/cross-platform-cryptography.md) that FriendlyName(s) are not supported in Unix.

provided you try to use Friendly Names in Unix-based systems it will result in this:
System.PlatformNotSupportedException : The FriendlyName value cannot be set on Unix.

**Comment by wfurt** [comment]
Is this documentation bug @bartonjs ? It seems like we would have to invent some metadata to store it outside of basic x509 data.

**Comment by bartonjs** [comment]
@wfurt Yeah, there's no plan to support it outside of Windows.  Not sure if Cross-Platform Cryptography is the right doc (since it's about higher level concepts), probably the docs.microsoft.com content.
